examples/simple/io_read_dumpfile.py

```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env python
+
+# type: ignore
```

Comment:
It was previous fix , but after adding `--follow-imports=skip` , it should not be needed now. thanks:)